src/app/tracing/dialog/dialog-ok-cancel/dialog-ok-cancel.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+export interface DialogOkCancelData {
+    title: string;
+    content1: string;
+    content2?: string;
+    cancelText: string;
+    okText: string;
+}
+
+@Component({
+    selector: 'fcl-dialog-ok-cancel',
+    templateUrl: './dialog-ok-cancel.component.html',
+    styleUrls: ['./dialog-ok-cancel.component.scss']
+})
+export class DialogOkCancelComponent {
+    dialogData: DialogOkCancelData;
+
+    constructor(
+        public dialogRef: MatDialogRef<DialogOkCancelComponent>,
+        @Inject(MAT_DIALOG_DATA) public data: DialogOkCancelData
+    ) {
+        this.dialogData = data;
+    }
+
+    cancelDialog() {
+        this.dialogRef.close(this.data.cancelText);
+    }
+
```

Comment:
Please change function name
Please use Constants.DIALOG_OK rather then this.data.okText (to be more agnostic)

Reply:
I done in Ticket #669